pytorch_lightning/accelerators/accelerator.py

```diff
@@ -338,21 +338,42 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         return self.training_type_plugin.lightning_module_state_dict()
 
     def barrier(self, name: Optional[str] = None) -> None:
+        """
+        .. deprecated:: v1.5
+            This method is deprecated in v1.5 and will be removed in v1.6.
+            Please call training_type_plugin.barrier directly
+        """
+        rank_zero_deprecation(
+            "This method is deprecated in v1.5 and will be removed in v1.6."
+            "barrier logic is implemented directly in the :class:`TrainingTypePlugin` implementations."
+        )
         self.training_type_plugin.barrier(name=name)
 
     def broadcast(self, obj: object, src: int = 0) -> object:
         """Broadcasts an object to all processes, such that the src object is broadcast to all other ranks if
         needed.
 
+        .. deprecated:: v1.5
+            This method is deprecated in v1.5 and will be removed in v1.6.
+            Please call training_type_plugin.broadcast directly
+
         Args:
             obj: Object to broadcast to all process, usually a tensor or collection of tensors.
             src: The source rank of which the object will be broadcast from
         """
+        rank_zero_deprecation(
+            "This method is deprecated in v1.5 and will be removed in v1.6."
```

Comment:
Readers of these messages won't know what "this method" refers to. Can you state the name explicitly?